cert/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/lightningnetwork/lnd/cert
 
 go 1.13
+
+require github.com/stretchr/testify v1.6.1
```

Comment:
This should also have a `go.sum` with it? Also I think we can leave testify at v1.5.1 (which is what lnd has, and is why the linter is breaking rn). Modules always try to bump to the latest version, so you can just manually edit and mod tidy again.

Reply:
Whoa, not sure what happened there. Sorry about that! I added in the go.sum and also set testify to 1.5.1.

Reply:
Modules have a mind of their own :D When you don't set a version they just go for the latest one, can be annoying sometimes because bumps stuff without you noticing.